webui/react/src/components/PageHeaderFoldable.tsx

```diff
@@ -79,14 +102,7 @@ const PageHeaderFoldable: React.FC<Props> = ({ foldableContent, leftContent, opt
           <div className={css.optionsButtons}>
             {options?.slice(0, 3).map((option) => (
               <div className={css.optionsMainButton} key={option.key}>
-                <Button
-                  disabled={option.disabled || !option.onClick}
-                  icon={option?.icon}
-                  key={option.key}
-                  loading={option.isLoading}
-                  onClick={option.onClick}>
-                  {renderOptionLabel(option)}
-                </Button>
+                {option.content}
```

Comment:
Some styles needed 
<img width="339" alt="Screenshot 2023-12-05 at 3 08 06 PM" src="https://github.com/determined-ai/determined/assets/40620519/40cf8b40-9d13-476c-ab78-241201113c89">

Reply:
@gt2345 what styling is needed here exactly?

Reply:
To be precise, our current implementation looks like this
<img width="346" alt="Screenshot 2023-12-05 at 3 17 30 PM" src="https://github.com/determined-ai/determined/assets/40620519/4a4e4f23-4e9f-4245-a5c0-46c238bed766">

Reply:
fixed!